e2e/testsuite/testsuite.go

```diff
@@ -399,6 +400,22 @@ func (*E2ETestSuite) TransferChannelOptions() func(options *ibc.CreateChannelOpt
 	}
 }
 
+// FeeMiddlewareChannelOptions configures both of the chains to have fee middleware enabled.
+func (s *E2ETestSuite) FeeMiddlewareChannelOptions() func(options *ibc.CreateChannelOptions) {
```

Comment:
I took the liberty to move this function here.

Reply:
Thank you @crodriguezvega!